erizo/src/erizo/WebRtcConnection.cpp

```diff
@@ -216,13 +209,11 @@ namespace erizo {
     bool res = false;
     if(tempSdp.initWithSdp(sdp, theMid)){
       if (theType == VIDEO_TYPE||bundle_){
-        ELOG_DEBUG("Setting VIDEO CANDIDATE" );
         res = videoTransport_->setRemoteCandidates(tempSdp.getCandidateInfos(), bundle_);
       } else if (theType==AUDIO_TYPE){
-        ELOG_DEBUG("Setting AUDIO CANDIDATE");
         res = audioTransport_->setRemoteCandidates(tempSdp.getCandidateInfos(), bundle_);
       }else{
-        ELOG_ERROR("Cannot add remote candidate with no Media (video or audio)");
+        ELOG_ERROR("%s msg:Cannot add remote candidate with no Media (video or audio), candidate: %s", toLog(), sdp.c_str() );
```

Comment:
I'd rather use message as in other logs:
`ELOG_ERROR("%s, message: add remote candidate with no Media (video or audio), candidate: %s", toLog(), sdp.c_str() );`